src/systems/ec/implementations/vanilla/packages/ec-utility-typography/ec-utility-typography.scss

```diff
@@ -140,6 +140,11 @@
       .ecl-u-type-color-#{$key} {
         color: $value;
       }
+
+      .ecl-u-type-color-hover-#{$key}:hover,
+      .ecl-u-type-color-hover-#{$key}:focus {
```

Comment:
shouldn't it be `ecl-u-type-color-focus-#{$key}:focus`? what about `:active`? can't it also be coupled with `.ecl-u-type-color-#{$key}` (since the declaration is the same)

Reply:
let's keep it simple: one css class for normal color, and one for pseudo-class color (hover, focus and active). So I guess the class should be renamed to `ecl-u-type-color-alt-*` or somthing like that.
It is already an edge case, Idoubt we need to deeper into details